src/services/services.ts

```diff
@@ -7866,6 +7886,30 @@ namespace ts {
             return [];
         }
 
+        function getCodeFixesAtPosition(fileName: string, start: number, end: number, errorCodes: string[]): CodeAction[] {
+            synchronizeHostData();
+            const sourceFile = getValidSourceFile(fileName);
+            const checker = program.getTypeChecker();
+            let allFixes: CodeAction[] = [];
+
+            forEach(errorCodes, error => {
+                const context = {
+                    errorCode: error,
+                    sourceFile: sourceFile,
+                    span: { start, length: end - start },
+                    checker: checker,
+                    newLineCharacter: getNewLineOrDefaultFromHost(host)
+                };
+
+                const fixes = codeFixProvider.getFixes(context);

```

Comment:
use ts.concatenate instead:

```ts
allFixes = concatenate(allFixes , codeFixProvider.getFixes(context));
```

alternatively, you can rewrite this foreach as a reduce call.